packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/{{#cicd_github}}.github{{/cicd_github}}/workflows/build_and_deploy_android_app.yaml

```diff
@@ -46,13 +46,17 @@ jobs:
     env:
       CREDENTIAL_ENCRYPTION_PASSWORD: ${{ secrets.credential_encryption_password }}
       CREDENTIAL_REPOSITORY_ACCESS_SECRET: ${{ secrets.credential_repository_access_secret }}
-      BUNDLE_GEMFILE: ${{ inputs.project_path }}/Gemfile
     steps:
       - uses: actions/checkout@v3
       - uses: subosito/flutter-action@v2
         with:
           flutter-version: ${{ inputs.flutter_version }}
           channel: "stable"
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
```

Comment:
Setting java to version 17 isn't always necessary, in the case of todoapp it was due to dependencies requirement. 
So could this cause potential problems in the future?

Reply:
Fair question. From my tests, the builds were working fine with the java version set to `17`. 

Also, projects generated with the `flutter create` command (which we use internally in `rx_bloc_cli`) come preconfigured with java version `1.8`. Updating it to `17` should also work as it is backwards compatible.

The `setup-java` action installs the specific version on the machine and adds it to its path. So that should resolve any problems in case some machine doesn't have it installed.